### PR TITLE
Sa openshift

### PR DESCRIPTION
**Pull request by alinadonisa**

### CODE REVIEW

File: pkg/condition/condition.go [hunk 1]
```diff
@@ -65,6 +65,9 @@ func SetFalse(ctype api.ClusterConditionType, status *api.CrdbClusterStatus, now
 func SetTrue(ctype api.ClusterConditionType, status *api.CrdbClusterStatus, now metav1.Time) {
 	setStatus(ctype, metav1.ConditionTrue, status, now)
 }
+func SetOperatorCondTrue(ctype api.ClusterConditionType, status *api.CrdbClusterStatus, now metav1.Time) {
```
Comment (chrislovecnm) [inline pkg/condition/condition.go:68]: Need godoc

File: pkg/controller/cluster_controller.go [hunk 2]
```diff
@@ -89,6 +89,13 @@ func (r *ClusterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 
 	cluster := resource.NewCluster(cr)
+	// //fix for open shift
+	cluster.SetOperatorCondTrue(api.InitOperatorCondition)
+	if err := r.Client.Status().Update(ctx, cluster.Unwrap()); err != nil {
+		log.Error(err, "failed to update cluster status on InitOperatorCondition")
+		//return requeueIfError(err)
```
Comment (chrislovecnm) [inline pkg/controller/cluster_controller.go:96]: remove dead code

File: pkg/features/operator_features.go [hunk 3]
```diff
@@ -51,4 +55,5 @@ func init() {
 var defaultOperatorFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	PartitionedUpdate: {Default: false, PreRelease: featuregate.Alpha},
 	Decommission:      {Default: false, PreRelease: featuregate.Alpha},
+	Upgrade:           {Default: true, PreRelease: featuregate.Alpha},
```
Comment (chrislovecnm) [inline pkg/features/operator_features.go:58]: @alinadonisa can we deco Upgrade and Enable the others?  I do not want to push a PR that will conflict with yours

File: test-registry.yaml [hunk 4]
```diff
@@ -19,4 +19,4 @@ metadata:
 spec:
   sourceType: grpc
   displayName: Cockroach Test Demo
-  image: PLACEHOLDER
+  image: quay.io/alinalion/cockroachdb-operator-index:1.1.21
```
Comment (chrislovecnm) [inline test-registry.yaml:22]: can we do a placeholder here?

**Review by chrislovecnm — CHANGES_REQUESTED** [review]
Looks good, just nitpicks

File: pkg/testutil/fake.go [hunk 5]
```diff
@@ -206,9 +206,21 @@ func (c *FakeClient) DeleteAllOf(_ context.Context, obj runtime.Object, opts ...
 }
 
 func (c *FakeClient) Status() client.StatusWriter {
-	panic("implement me")
+	// panic("implement me")
```
Comment (chrisseto) [inline pkg/testutil/fake.go:209]: nit: remove the commented out panic line.

**Review by chrisseto — APPROVED** [review]
LGTM with the smallest if nits.

**Review by chrisseto — APPROVED** [review]

**Review by chrislovecnm — APPROVED** [review]

**Review by chrislovecnm — APPROVED** [review]

**Pull request merged**